.github/workflows/builds.yml

```diff
@@ -2489,12 +2496,20 @@ jobs:
                 linux-headers-$(uname -r)
 
       - name: Checkout Netmap repository
+        if: steps.netmap-cache.outputs.cache-hit != 'true'
         uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938
         with:
           repository: luigirizzo/netmap
           # gets cloned to $GITHUB_WORKSPACE/netmap/
           path: netmap/
 
+      - name: Save Netmap Cache
```

Comment:
You could also cache the compilation

Reply:
I don't want to add additional complexity, this is just about avoiding the rate limit issue